packages/ai/tests/ai/modules/task/test_env_var_exfil.py

```
@@ -0,0 +1,253 @@
+from unittest.mock import patch
+
+
+class _FakeTask:
+    """
+    Minimal stand-in that carries only _resolve_pipeline and ALLOWED_ENV_PREFIX
+    from the real Task class, avoiding heavyweight __init__ dependencies.
+    """
+
+    ALLOWED_ENV_PREFIX = 'ROCKETRIDE_'
+
+    def _resolve_pipeline(self, pipeline):
```

Ruff (0.15.7)

[warning] Missing return type annotation for private function `_resolve_pipeline`

(ANN202)